summary/store/summary.go

```
@@ -49,6 +49,137 @@ func NewTypeless(delegate *storeStructuredMongo.Repository) *TypelessSummaries {
+	selector := bson.M{"dates.outdatedSince": bson.M{"$lte": time.Now().UTC()}}
+
+	opts := options.Find()
+	opts.SetSort(bson.D{{Key: "dates.outdatedSince", Value: 1}})
+	opts.SetLimit(int64(limit))
+	opts.SetProjection(bson.M{"userId": 1, "type": 1, "dates.outdatedSince": 1})
+
+	cursor, err := r.Find(ctx, selector, opts)
+	if err != nil {
+		return nil, fmt.Errorf("unable to list outdated summaries: %w", err)
+	}
+	defer cursor.Close(ctx)
```

golangci-lint (2.12.2)

[error] Error return value of `cursor.Close` is not checked

(errcheck)